bftengine/src/bftengine/messages/PrePrepareMsg.hpp

```diff
@@ -68,6 +69,14 @@ class PrePrepareMsg : public MessageBase {
                 const concordUtils::SpanContext& spanContext,
                 size_t size);
 
+  PrePrepareMsg(ReplicaId sender,
+                ViewNum v,
+                SeqNum s,
+                CommitPath firstPath,
+                const concordUtils::SpanContext& spanContext,
+                size_t size,
+                const std::string& correlationID);
```

Comment:
let's call this `batchCid` or `batch_cid` everywhere in this PR

Reply:
Fixed.